src/openloco/windowmgr.cpp

```diff
@@ -14,6 +14,45 @@ namespace openloco::ui::windowmgr
         constexpr uint16_t by_type = 1 << 7;
     }
 
+    class Container
```

Comment:
I'm not sure I see the need for such a class. Can we make do with a `std::vector<window*> foo(_windows, _windows_end);` instead?

Reply:
@AaronVanGeffen You can't use `std::vector<window*>`, but you could do `std::array<window>*` pointing to the address of where the windows are.